tests/apollo/test_skvbc_preexecution.py

```diff
@@ -191,3 +192,41 @@ async def test_view_change(self, bft_network):
                 break
 
         self.assertEqual(new_last_block, last_block)
+
+    @with_trio
+    @with_bft_network(start_replica_cmd)
+    async def test_f_nonprimary_crash_amid_parallel_tx(self, bft_network):
+        '''
+        Crash f nonprimary replicas while multiple transactions are being processed.
+        Block processing of the network should be unaffected with f-count interruption.
+        Check created block lengths match submitted x count of transactions.
+        '''
+        bft_network.start_all_replicas()
+        skvbc = kvbc.SimpleKVBCProtocol(bft_network)
+        f_count = int((len(bft_network.all_replicas()) - 1) / 3)
+        primary_index = 0
+
+        await trio.sleep(2)
+        
+        num_of_requests = 20
+        clients = bft_network.random_clients(1)
+        reader_client = list(clients)[0]
+        nonprimaries = bft_network.all_replicas(without={primary_index})
+        crash_targets = random.sample(nonprimaries, f_count) # pick random f to crash
+        
+        started_block_count = skvbc.parse_reply(await reader_client.read(skvbc.get_last_block_req()))
+        with trio.move_on_after(seconds=20): # around tx_count seconds for enough buffer time
+            async_requests = self.run_concurrent_pre_execution_requests(
+                                            skvbc, clients, num_of_requests,
+                                            write_weight=1.0, interval=0.05)
```

Comment:
Why specifically `interval=0.05`?

Reply:
Could be 0.01 or even lower. I just wanted parallel preexecution transactions to submit continuously so that it is more likely to crash replicas in the middle of processing a tx.

Reply:
Should I revert adding `interval` option? The test won't be mid-tx crash anymore so I don't really need to control this.

Reply:
Yes, let's revert this. Thanks!